README.md

```diff
@@ -111,6 +111,93 @@ We upload to two channels:
 - `stable`: this channel has all the stable packages.
 - `latest`: this channel will have the latest packages in development/test phase (including stable releases).
 
+To find all the stable versions uploaded for a particular package, try:
+
+```
+    $ conan search -r includeos '<package_name>/*@includeos/stable'
+```
+> **NOTE:** We only guarantee that the **latest 10 packages** are kept in the
+`latest` channel. All `stable` packages will be kept in the stable channel unless
+proven unsafe. One suggested workaround is to copy packages into your own repository.
+YOu can do that using the copy command:
+
+```
+    $ conan copy --all -p <path-to-conanfile> <package_name>@<your_repository>/<your_channel>
+```
+___
```

Comment:
The part above of the README has changed on the dev branch. I think you have merged in some old changes.